ui/app/src/components/shared/CostsTag.tsx

```diff
@@ -0,0 +1,26 @@
+import { Stack } from "@fluentui/react";
+import { useContext } from "react";
+import { CostsContext } from "../../contexts/CostsContext";
+
+interface CostsTagProps {
+  resourceId: string
+}
+
+export const CostsTag: React.FunctionComponent<CostsTagProps> = (props: CostsTagProps) => {
+  const costsCtx = useContext(CostsContext);
+  const resourceCosts = costsCtx?.costs?.find((resourceCost) => {
+    return resourceCost.id === props.resourceId;
+  });
+  let costBadge = <></>;
+  if (resourceCosts) {
```

Comment:
you might have a bug here when there are no costs available? I'm seeing a page failure on Shared-Services, with the console log:
![image](https://user-images.githubusercontent.com/35696285/200809613-adb6f137-b815-4900-ab42-4dfbba1b08aa.png)

Reply:
Ah good catch, I shall look at that now thanks!

Reply:
Think I've fixed it but can't test as I have costs available - could you pull and check?

Reply:
yup, looks good!